publisher/server_only/README.md

```diff
@@ -15,17 +15,19 @@ docker build . -t uid2-publisher-server
 docker run -it --rm -p 3000:3000 \
     -e UID2_BASE_URL="https://integ.uidapi.com" \
```

Comment:
Should this default point to our integration URL? That's https://operator-integ.uidapi.com. Hopefully people won't be using production keys to try out the example app.